src/main/kotlin/com/terraformation/backend/search/table/SpeciesGrowthFormsTable.kt

```diff
@@ -0,0 +1,37 @@
+package com.terraformation.backend.search.table
+
+import com.terraformation.backend.db.default_schema.SpeciesId
+import com.terraformation.backend.db.default_schema.tables.references.SPECIES
+import com.terraformation.backend.db.default_schema.tables.references.SPECIES_GROWTH_FORMS
+import com.terraformation.backend.search.SearchTable
+import com.terraformation.backend.search.SublistField
+import com.terraformation.backend.search.field.SearchField
+import org.jooq.Record
+import org.jooq.SelectJoinStep
+import org.jooq.TableField
+
+class SpeciesGrowthFormsTable(private val tables: SearchTables) : SearchTable() {
+  override val primaryKey: TableField<out Record, out Any?>
+    get() = SPECIES_GROWTH_FORMS.SPECIES_ID
```

Comment:
Because this table has a compound primary key, you'll need to tell jOOQ to create a synthetic column that combines the species and growth form IDs, and use that column as the primary key here. At the bottom of `Config.kt`, you'll find a variable `EMBEDDABLES` that has a bunch of existing examples you can follow.

The reason we need this is because of what the primary key is used for in the search API. Basically it creates a SQL query that's structured like
```sql
SELECT field1, field2, field3
FROM table_from_search_prefix
WHERE primary_key IN (
    SELECT primary_key
    FROM table_from_search_prefix
    WHERE conditions_from_search_request
)
```
which only works if the primary key is guaranteed to refer to a single row. With a compound primary key, if you only use one of the columns, that query will return more results than expected.

Reply:
CC @tommylau523 on the above since you're adding a compound primary key to a table too (though your table isn't exposed via the search API yet).